### PR TITLE
Add changelog in prepartion for 0.2.0 release

### PR DESCRIPTION
**Pull request by aarondav**
Copying the format [from Kubernetes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.9.md) for attributing major contributions.

### CODE REVIEW

**Review by smurching — COMMENTED** [review]
One comment but otherwise this looks good, thanks @aarondav

**Comment by smurching** [comment]
Also cc @tomasatdatabricks, are we missing anything on the models side? e.g maybe we should mention the newly-exposed model deployment Python APIs in #36?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+=========
+
+0.2.0 (2018-06-27)
+------------------
+
+- Added ``mlflow server`` to provide a remote tracking server. This is akin to ``mlflow ui`` with new options:
+    - ``--host`` to allow binding to any ports (#27, @mdagost)
+    - ``--artifact-repo`` to allow storing artifacts at a remote location, S3 only right now (#78, @mateiz)
+    - Server now runs behind gunicorn to allow concurrent requests to be made (#61, @mateiz)
+- Tensorflow support, we now support logging Tensorflow Models directly in the log_artifacts API, Model format, and serving APIs. This also works for Keras. (#28, @juntai-zheng)
```
Comment (mateiz) [inline CHANGELOG.rst]: I'd say "TensorFlow integration" since it was possible to use TF earlier using the more generic APIs.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- Added ``experiments.list_experiments`` as part of experiments API (#37, @mparkhe)
+- Improved support for unicode strings (#79, @smurching)
```
Comment (mateiz) [inline CHANGELOG.rst]: We should also add "miscellaneous bug and documentation fixes"; there were a few of these. And we might want to mention the diabetes dataset example from Denny + the notebook version of that if we merge that.

**Review by mateiz — COMMENTED** [review]
Looks OK to me. One question is whether it will get annoying to list the PR and author on every issue in the future, but maybe it's ok to leave it for now. It might just make it tougher to generate a changelog but it is nice for reference.

**Comment by aarondav** [comment]
The format of the CHANGELOG is one of the most reversible decisions in the history of mankind, so I wouldn't worry too much if it becomes a burden to keep the contributors and PRs linked.

**Comment by aarondav** [comment]
All comments addressed.

**Review by smurching — APPROVED** [review]
LGTM :)

**Review by mateiz — APPROVED** [review]

**Comment by mateiz** [comment]
Looks good to me too.

**Pull request merged**